modules/olmv1-catalog-queries.adoc

```diff
@@ -20,64 +46,64 @@ You can query catalogs by using the `jq` CLI tool.
 a|
 [source,terminal]
 ----
-$ jq -s '.[] \| select( .schema == "olm.package") \| \
-  .name' <catalog_name>.json
+$ opm render <catalog_registry_url>:<tag> \
+  \| jq -s '.[] \| select( .schema == "olm.package")
 ----
 
 |Packages that support `AllNamespaces` install mode and do not use webhooks
 a|
 [source,terminal]
 ----
-$ jq -c 'select(.schema == "olm.bundle") \| \
-  {"package":.package, "version":.properties[] \| \
-  select(.type == "olm.bundle.object").value.data \| \
-  @base64d \| fromjson \| \
-  select(.kind == "ClusterServiceVersion" and (.spec.installModes[] \| \
-  select(.type == "AllNamespaces" and .supported == true) != null) \
-  and .spec.webhookdefinitions == null).spec.version}' \
-  <catalog_name>.json
+$ opm render <catalog_registry_url>:<tag> \
+  \| jq -cs '[.[] \| select(.schema == "olm.bundle" and (.properties[] \
+  \| select(.type == "olm.csv.metadata").value.installModes[] \
+  \| select(.type == "AllNamespaces" and .supported == true)) \
+  and .spec.webhookdefinitions == null) \
+  \| .package] \| unique[]'
 ----
 
 |Package metadata
 a|
 [source,terminal]
 ----
-$ jq -s '.[] \| select( .schema == "olm.package") \| \
-  select( .name == "<package_name>")' <catalog_name>.json
+$ opm render <catalog_registry_url>:<tag> \
+  \| jq -s '.[] \| select( .schema == "olm.package") \
+  \| select( .name == "<package_name>")'
 ----
 
 |Catalog blobs in a package
 a|
 [source,terminal]
 ----
-$ jq -s '.[] \| select( .package == "<package_name>")' \
-  <catalog_name>.json
+$ opm render <catalog_registry_url>:<tag> \
+  \| jq -s '.[] \| select( .package == "<package_name>")'
 ----
 
 |===
 
 .Common channel queries
-[options="header",cols="1,3"]
+[options="header",cols="1,3"]]
```

Comment:
I'm not familiar with `adoc` format but shouldn't this be a single closing bracket `]`?
```suggestion
[options="header",cols="1,3"]
```

Reply:
Good catch!

Reply:
@dtfranz TY! Addressed.